.gitlab-ci.yml

```diff
@@ -16,6 +16,7 @@ variables:
   SSTATE_LOCATION: "${RUNNER_SSTATE}/${CI_PROJECT_NAME}"
   SSTATE_MIRRORS: "file://.* ${SSTATE_LOCATION}/PATH;downloadfilename=PATH"
   SSTATE_MAX_AGE: "3d"
+  DISTRO_APT_PREMIRRORS: 'http://deb\.debian\.org https://debian.inf.tu-dresden.de\n'
```

Comment:
If that only affects our CI setup, then better make this a CI variable that is using this value only in our local project settings.